content/administration/upgrade.rst

```diff
@@ -1,12 +1,252 @@
+      team. This will slow down the processing and response time.
+
+Please provide as much detail as you can (i.e., videos and screenshots to illustrate your issue).
+This will avoid clarifying questions and speed up the resolution process significantly.
+
+.. note::
+   * The purpose of the test phase is not to correct existing data or configurations in your
+     database.
+   * |assistance-contact|
+
+.. _upgrade/steps-production:
+
+The production launch
+=====================
+
+The production upgrade request is when you decide to upgrade your current database with all your
+production data (invoices, VAT returns, inventories, current orders) to a new version of your
+choice.
+
+After your :ref:`tests <upgrade/testing-phase>` are completed to your satisfaction, submit the
+request to upgrade your production database via our `website form <https://upgrade.odoo.com>`_.
+Select *Production* purpose.
+
+.. important::
+   Going into production without first testing may lead to:
+
+   - business interruptions (e.g., no longer having the possibility to validate an action)
+   - poor customer experiences (e.g., an eCommerce website that does not work correctly)
+
+.. _upgrade/production-assistance:
+
+Assistance
+----------
+
+If you encounter issues or problems in the **production database**, please get in touch with **Odoo
+Support**:
+
+#. Connect to our `Odoo Support page <https://www.odoo.com/help>`_.
+#. Under the *Ticket Description* section, select the appropriate type related to your issue but
+   **do not select** the option *An issue related to my upgrade*.
+
+   .. note::
+      After upgrading to production, the support will be provided by the Support team instead of the
+      Upgrade team.
+
+#. Please provide as much detail as you can (i.e., videos and screenshots to illustrate your issue).
+   This will avoid clarifying questions and speed up the resolution process significantly.
+
+   .. warning::
+      If you choose *An issue related to my upgrade* as ticket type, the request will be redirected
+      to another team than the support one and will slow down the processing and response time.
+
+.. _upgrade/assistance:
+
+Help
+====
+
+.. _upgrade/contact:
+
+Contact our Upgrade service support
+-----------------------------------
+
+Should you have any more questions about the upgrade, do not hesitate to send a message to `Odoo
+Upgrade Team <mailto:upgrade@odoo.com>`_. We will be happy to answer it as soon as possible.
+
+.. _upgrade/supported-versions:
+
+Supported versions
+------------------
+
+Please note that Odoo provides support and bug fixing only for the three last major versions of
+Odoo.
+
+This is a factor to take into consideration before upgrading. If you are on an older version, we
+suggest you to prefer the most recent version to benefit from longer support (before having to
+upgrade again).
+
+.. seealso::
+   :doc:`maintain/supported_versions`.
```

Comment:
Is the dot intended here?
```suggestion
   :doc:`maintain/supported_versions`
```
Also, it usually looks better and requires less code when adding a new item to systematically use bulleted lists in See Also directives:
```suggestion
   - :doc:`maintain/supported_versions`
```
Not mandatory, though!